libs/data-mapper/src/lib/models/Function.ts

```diff
@@ -512,5 +513,45 @@ export const functionMock: FunctionData[] = [
     category: FunctionCategory.Collection,
     description: 'Sort the sequence by a given property',
   },
+  {
+    key: 'XPath',
+    maxNumberOfInputs: 1,
+    functionName: 'xpath',
+    outputValueType: NormalizedDataType.Any,
+    inputs: [
+      {
+        name: 'XPATH expression',
+        allowedTypes: [NormalizedDataType.String],
+        displayAttribute: DisplayAttribute.TextBox,
```

Comment:
Might want to add in some tests that test things like. Multiple input and other like that to make sure we are serializing the data in and out correctly

Reply:
I'll do this on the next PR, I need to do some similar work with XML files anyways